rts/Game/UI/MouseHandler.cpp

```diff
@@ -619,7 +619,7 @@ std::string CMouseHandler::GetCurrentTooltip() const
 		std::string s;
 
 		if (luaInputReceiver->IsAbove(lastx, lasty)) {
-			s = std::move(luaInputReceiver->GetTooltip(lastx, lasty));
+			s = luaInputReceiver->GetTooltip(lastx, lasty);
```

Comment:
I'd check here and below if the compilers do copy elision.
If I were to suspect a case when NVRO might not kick in, it'd be that function.

Reply:
In that case `std::move` would still do nothing because the value is already an xvalue though?